README.md

```diff
@@ -62,7 +62,7 @@ TODO
 
 Use the following commands for local development:
 
-- `npm start` - TODO
+- `npm start` - Starts dev server on http://localhost:8089/
```

Comment:
Can you also update the setup section please? It's got a couple of todos in there